ui/page/creatorMemberships/supporterArea/index.js

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
```

Comment:
:small_red_triangle: Donor Portal not populated on sign in

- http://localhost:9090/$/memberships/supporter and then sign out.
- Sign In from here so that it redirects back to http://localhost:9090/$/memberships/supporter
- I get a blank overview, and I think the CSS isn't loaded correctly, because in another account with no donations, the "Find creators..." sentence was in red. (@toshokanneko, load order problem?)
    - :x:  <img width="555" alt="image" src="https://user-images.githubusercontent.com/64950861/191066797-64cd6df1-4a09-4f95-b50f-1a3f3483587a.png">
- After refreshing again, then data appears.
    - :heavy_check_mark: <img width="556" alt="image" src="https://user-images.githubusercontent.com/64950861/191066938-c0b15b37-d54e-4850-a846-28f1b236a05b.png">

Reply:
Yeah, can't reproduce reliably either.  Sometimes ok, sometimes not.  But probably no big deal.

<img width="303" alt="image" src="https://user-images.githubusercontent.com/64950861/191088365-d9970eec-abd7-42f8-8d4a-b7aada46e397.png">

Reply:
Jep, it's wrapped in the `bank-account-status` class for whatever reason. Probably the result of copy / paste.

Reply:
It's now wrapped in `p` without an attached class.